CHANGES.md

```diff
@@ -1,3 +1,58 @@
+### Unreleased
+
+#### Parser (2021-01-??)
+
+Replace Earley by Menhir, Pratter and Sedlex
+
+**Syntax modifications:**
+
+- Commands and tactics must now be ended by a semi-colon `;`.
+
+- The syntax `λx y z: nat, ...` with multiple variables is not
+  authorised anymore, but `λ(x y z: nat), ...` is, as well as `λ x :
+  N, t` with a single variable.
+
+- In unification rules, the right hand-side must now be enclosed in
+  `begin ... end`, so
+  ```
+  set unif_rule $x + $y ≡ 0 ↪ $x ≡ 0; $y ≡ 0
+  ```
+  becomes
+  ```
+  set unif_rule $x + $y ≡ 0 ↪ begin $x ≡ 0; $y ≡ 0 end;
```

Comment:
Finally, I realize that using begin/end for unification rules interferes with the navigation commands in VSCode and Emacs where begin is used to jump to the next/previous proof. So, we should use something else. What did you propose in the first place? Curly brackets? Square brackets?